tests/models/language.py

```diff
@@ -28,3 +34,64 @@ def test_language_specialchars_uniqueness():
     language.specialchars = u" Čč Ḍḍ Ɛɛ Ǧǧ Ɣɣ  Ḥḥ Ṣṣ ṬṬṭ Ẓẓ Ţţ Ṛṛ‌Ṛṛ‌"
     language.save()
     assert language.specialchars == u" ČčḌḍƐɛǦǧƔɣḤḥṢṣṬṭẒẓŢţṚṛ‌"
+
+
+@pytest.mark.django_db
+def test_language_iso_fullname(afrikaans):
+    new_fullname = "FULLNAME"
+    fullname = afrikaans.fullname
+    afrikaans.fullname = new_fullname
+    afrikaans.save()
+    assert afrikaans.fullname == new_fullname
+
+    settings.LANGUAGE_CODE = "cy"
+    ui_lang = to_locale(settings.LANGUAGE_CODE)
+    with translation.override(ui_lang):
```

Comment:
i think this would be a lot easier to read if it was

```
with translation.override("cy"):
   ...
```
ui_lang is not used, and you could get rid of the actual_ which i think would simplify